Demo/EarlGreyExample/EarlGreyContribSwiftTests/GetText.swift

```diff
@@ -0,0 +1,22 @@
+/*
+ *  Example Usage:
+ *
+ *  let element = Element()
+ *
+ *  domainField.performAction(grey_typeText("hello.there"))
+ *             .performAction(grey_getText(element))
+ *
+ *  GREYAssertTrue(element.text != "", reason: "get text failed")
+ */
+func grey_getText(elementCopy: Element) -> GREYActionBlock {
```

Comment:
> Is there any reason why these are Swift only classes?

Yes, I have a growing collection of Swift examples. There are already a bunch of examples for Objective C. Before the code would just be in the markdown file, instead now they're in a test target that properly validates the functionality of the examples.

> We can add these to GREYActions or so and have them exported in the exact same way that all other matchers are.

That's the long term plan ([for example: grey_replaceText](https://github.com/google/EarlGrey/pull/119)). I expect grey_firstElement will be replaced by an [nth element matcher](https://github.com/google/EarlGrey/issues/138) in core once that's implemented.

`grey_getText` is new and it seemed safer to let people try it out first before adding a similar matcher to EarlGrey core. Ideally all the matchers would end up in `GREYActions` once they've proven useful.

> Also, I'm not completely sure we need an entirely new project for this.

To clarify, this is a new testing target on an existing project -- not a new project. 

> In the FunctionalTests.xcodeproj, could we not add the view controller and tests for these methods with a separate of "EarlGreyExperimentalTests.m" file or so?

`FunctionalTests` test EarlGrey itself and the new matchers aren't intended to be accepted into EarlGrey yet. If it's fine to just add the experimental matchers directly to core and test them like all the other features, then `FunctionalTests` makes sense. Do we have a way to mark APIs as experimental? I expect them to change (potentially quite a bit).